### DISCUSSION TITLE
init of tiktok client is stuck when code is run on amazon cloud(locally its working great)

### DISCUSSION THREAD

**Discussion by Daniel-Verman** [davidteather/TikTok-Api#1008] [Q&A]
Hi im trying to use this code to extract stats from video_id.
and its working great locally,
but when im uploading the code to aws to run on the cloud(its on airflow dag) its stuck when trying to init the tiktok client, i 
tried different variation with:

tk_api = TikTokApi() # its not moving from this line
v_video = tk_api.video(id = v_id)
v_info = v_vido.info()

or

with TikTokApi() as tk_api:
     v_video = tk_api.video(id=v_id)
     v_info = v_video.info()

any idea why its working locally and not on amazon?is there anything i need to provide to 
TikTokApi params to make it work on amazon as well?

**Comment by davidteather** [comment]
What OS?
